src/popUp.js

```diff
@@ -0,0 +1,111 @@
+import { getMovieData, createComment } from './comments.js';
+import commentsCounter from './commentsCounter.js';
+
+// Pop up function
+const popUp = async (show) => {
+  const popUpContainer = document.createElement('article');
+  popUpContainer.id = 'pop-up';
+  popUpContainer.innerHTML = `
+    <div class="pop-container">
+       <div class="pop-up-header">
+        <h2 class="movie-title">${show.title}</h2>
+        <button id="close-modal-btn">x</button>
+       </div>
+      <div class="pop-up-body">
+        <div class="image-container">
+         <img src='${show.image.original}' class="movie-image">
+        </div>
+         <div class="pop-up-content">
+          <div class="left-content">
+            <label>Genre:</label>
+             <div class="movie-data">${show.genres[0]}</div>
+             <label>RunTime:</label>
+            <div class="movie-data">${show.runtime}</div>
+           </div>
+           <div class="right-content">
+             <label>Show Type:</label>
+             <div class="movie-data">${show.type}</div>
+             <label>Language:</label>
+             <div class="movie-data">${show.language}</div>
+           </div>
+         </div>
+       </div>
+       <div class="pop-up-comments">
+        <h3 id="comments-header"> Comments (0)</h3>
+        <ul id="comments-list">
+        </ul>
+      </div>
+      <div class="pop-up-form">
+        <h3>Add a comment</h3>
+        <form id="new-comment">
+          <input id="user" type="text" name="user" placeholder="Your name" required></input>
+          <textarea id="comment" name="user" placeholder="Add your comment" required></textarea>
+          <button type="submit" class="add-comment" id="comment-btn">Comment</button>
+        </form>
+      </div>
+     </div>
```

Comment:
- [ ] @HFG43 and @Kidd254 Great work adding the popup window, however someone cannot view the whole content in the popup without scrolling, kindly consider ensuring all content is visible like shown in the initial mockup, check the comparison below:


Mock-up Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/HFG43/JS_Capstone/assets/61408860/fec64ba5-1409-42a2-b332-b6b287b75faf) | ![image](https://github.com/HFG43/JS_Capstone/assets/61408860/fad13391-5ebf-46d9-abbf-34bfe8d987d2)